specification/cognitiveservices/data-plane/ComputerVision/v1.0/ComputerVision.json

```diff
@@ -0,0 +1,1482 @@
+                                "modelAsString": false
+                            },
+                            "enum": [
+                                "Celebrities",
+                                "Landmarks"
+                            ]
+                        }
+                    },
+                    {
+                        "name": "language",
+                        "in": "query",
+                        "description": "A string indicating which language to return. The service will return recognition results in specified language. If this parameter is not specified, the default value is &quot;en&quot;.Supported languages:en - English, Default.zh - Simplified Chinese.",
+                        "type": "string",
+                        "required": false,
+                        "default": "en",
+                        "x-nullable": "false",
+                        "x-ms-enum": {
+                            "name": "Language",
+                            "modelAsString": false
+                        },
+                        "enum": [
+                            "en",
+                            "zh"
+                        ]
+                    },
+                    {
+                        "$ref": "../../Common/Parameters.json#/parameters/ImageUrl"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The response include the extracted features in JSON format.Here is the definitions for enumeration typesClipartTypeNon-clipart = 0,  ambiguous = 1, normal-clipart = 2, good-clipart = 3.LineDrawingTypeNon-LineDrawing = 0,LineDrawing = 1.",
+                        "schema": {
+                            "$ref": "#/definitions/ImageAnalysis"
+                        }
+                    },
+                    "default": {
+                        "schema": {
+                            "$ref": "#/definitions/ComputerVisionError"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Successful Analyze with Url request": {
+                        "$ref": "./examples//SuccessfulAnalyzeWithUrl.json"
+                    }
+                }
+            }
+        },
+        "/generateThumbnail?overload=url": {
+            "post": {
+                "description": "This operation generates a thumbnail image with the user-specified width and height. By default, the service analyzes the image, identifies the region of interest (ROI), and generates smart cropping coordinates based on the ROI. Smart cropping helps when you specify an aspect ratio that differs from that of the input image. A successful response contains the thumbnail image binary. If the request failed, the response contains an error code and a message to help determine what went wrong.",
+                "operationId": "GenerateThumbnail",
```

Comment:
`operationId` must be `groupname_methodname` -- 

`groupname` is used to create a class in most generated languages
`methodname` is used to generate a method name in most languages